src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListUpdater.java

```diff
@@ -77,10 +78,14 @@ public void processDailyList(DailyListEntity dailyListEntity) throws JsonProcess
                         UserAccountEntity dailyListSystemUser = systemUserHelper.getDailyListProcessorUser();
                         hearing.setCreatedBy(dailyListSystemUser);
                         hearing.setLastModifiedBy(dailyListSystemUser);
+                        // set this so it's updated when no other changes are present
+                        hearing.setLastModifiedDateTime(OffsetDateTime.now());
```

Comment:
Consider injecting `CurrentTimeHelper` and using its `currentOffsetDateTime()` method to obtain the current time, instead of invoking the static method `OffsetDateTime.now()`.

This is a preferable approach as it allows us to mock the "current time" with a fixed value in unit tests.